src/storage/txn/commands/flashback_to_version_read_phase.rs

```diff
@@ -133,7 +135,7 @@ impl<S: Snapshot> ReadCommand<S> for FlashbackToVersionReadPhase {
                 if key_locks.is_empty() {
                     // No more locks to rollback, continue to the prewrite phase.
                     FlashbackToVersionState::Prewrite {
-                        key_to_lock: self.start_key.clone(),
+                        key_to_lock: start_key.clone(),
```

Comment:
This is not the exact key to lock anymore, maybe should rename it to `start_key`.